gotestcover.go

```diff
@@ -45,13 +48,32 @@ func run() error {
 	if err != nil {
 		return err
 	}
+
+	coverage := 0.0
+	coveredPackages := 0
+	coverageRe := regexp.MustCompile(`coverage: (\d+\.\d+)%`)
+
 	cov := runAllPackageTests(ps, func(out string) {
 		fmt.Print(out)
+
+		if flagCoverageSummary {
+			if m := coverageRe.FindStringSubmatch(out); m != nil {
+				c, _ := strconv.ParseFloat(m[1], 32)
+				coverage += c
```

Comment:
You can, there is a "+" operator defined for floats. You do that in games and in other applications all the time. This is not a finance application where it would matter. It simply does not matter that the coverage percentage is off, by... what? I do not want to even count what that is "0.30000000000000004"